openapi/listings/listings.yaml

```diff
@@ -148,16 +148,14 @@ components:
           scopes:
             listing: Access to the Listing Builder REST API
   schemas:
-    businessEntityListings:
-      title: Business Entity Listings
+    listingSyncListings:
+      title: Listing Sync Listings
       type: object
       x-lifecycle:
-        status: proposed
+        status: trustedTester
       description: |-
-        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
-        Gets a list of listings for a business entity
-      x-tags:
-        - Business Entity Listings
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        A list of listings for a business location
       x-internal: false
       properties:
         id:
```

Comment:
In the schema `listingSyncListings` you will need to update the `default` and `enum` options for the `type` field. It should match the path. (`default: listings` becomes `default: listingSyncListings`)

Reply:
Also they are missing from the `listingScores` schema.